src/city/ratings.c

```diff
@@ -316,90 +317,37 @@ static void update_culture_rating(void)
-        city_data.ratings.culture_points.religion = 9;
-    } else if (pct_religion > 30) {
-        city_data.ratings.culture_points.religion = 3;
-    } else {
-        city_data.ratings.culture_points.religion = 0;
-    }
-    city_data.ratings.culture += city_data.ratings.culture_points.religion;
-
-    int pct_school = city_culture_coverage_school();
-    if (pct_school >= 100) {
-        city_data.ratings.culture_points.school = 15;
-    } else if (pct_school > 85) {
-        city_data.ratings.culture_points.school = 10;
-    } else if (pct_school > 70) {
-        city_data.ratings.culture_points.school = 6;
-    } else if (pct_school > 50) {
-        city_data.ratings.culture_points.school = 4;
-    } else if (pct_school > 30) {
-        city_data.ratings.culture_points.school = 1;
-    } else {
-        city_data.ratings.culture_points.school = 0;
-    }
-    city_data.ratings.culture += city_data.ratings.culture_points.school;
-
-    int pct_academy = city_culture_coverage_academy();
-    if (pct_academy >= 100) {
-        city_data.ratings.culture_points.academy = 10;
-    } else if (pct_academy > 85) {
-        city_data.ratings.culture_points.academy = 7;
-    } else if (pct_academy > 70) {
-        city_data.ratings.culture_points.academy = 4;
-    } else if (pct_academy > 50) {
-        city_data.ratings.culture_points.academy = 2;
-    } else if (pct_academy > 30) {
-        city_data.ratings.culture_points.academy = 1;
-    } else {
-        city_data.ratings.culture_points.academy = 0;
-    }
-    city_data.ratings.culture += city_data.ratings.culture_points.academy;
-
-    int pct_library = city_culture_coverage_library();
-    if (pct_library >= 100) {
-        city_data.ratings.culture_points.library = 20;
-    } else if (pct_library > 85) {
-        city_data.ratings.culture_points.library = 14;
-    } else if (pct_library > 70) {
-        city_data.ratings.culture_points.library = 8;
-    } else if (pct_library > 50) {
-        city_data.ratings.culture_points.library = 4;
-    } else if (pct_library > 30) {
-        city_data.ratings.culture_points.library = 2;
-    } else {
-        city_data.ratings.culture_points.library = 0;
-    }
-    city_data.ratings.culture += city_data.ratings.culture_points.library;
-
+	
+    //ENTERTAINMENT: max 25
+    city_data.ratings.culture_points.average_entertainment_cp = (city_data.culture.average_theater + city_data.culture.average_amphitheater + city_data.culture.average_colosseum + city_data.culture.average_hippodrome) / 16;
+    city_data.ratings.culture += city_data.ratings.culture_points.average_entertainment_cp;
+
+    //SHOWS: unlimited
+    int total_shows = 0;
+    total_shows += city_entertainment_theater_shows();
+    total_shows += city_entertainment_amphitheater_shows();
+    total_shows += city_entertainment_colosseum_shows();
+    total_shows += city_entertainment_hippodrome_shows();
+    city_data.ratings.culture_points.total_shows_cp = total_shows;
```

Comment:
I'm guessing each show now contributes with 1 CP.

I'm not sure I like the idea. You can still "bomb" the city with 100 theaters and have 100 culture in the city.

Besides, if I'm not mistaken, only active (with shows) entertainment buildings spawn walkers, so you're effectively doubling the cultural weight of entertainment buildings here.

I like the idea of C4, it being that an active hippodrome adds 10 CP. Maybe keep that and distribute the rest of the remaining cultural element's weight to go to 90?

Also, we are planning a change to the colosseum (it will become a monument), so if it becomes a single building, maybe in the future the colosseum can be worth 10 CP and the hippodrome 15 CP, with the remaining elements being 25 CP each.